unpackaged/config/trial/profiles/Admin.profile

```diff
@@ -126,4 +126,12 @@
         <tab>hed__Application__c</tab>
         <visibility>DefaultOn</visibility>
     </tabVisibilities>
+    <tabVisibilities>
+        <tab>hed__Test__c</tab>
+        <visibility>DefaultOn</visibility>
+    </tabVisibilities>
+    <tabVisibilities>
+        <tab>hed__Test_Score__c</tab>
+        <visibility>DefaultOn</visibility>
+    </tabVisibilities>
```

Comment:
@MatthewBlanski  Since we are now using page layouts from Trial folder should we still have %%%NAMESPACE%%%  here for the page layouts? https://github.com/SalesforceFoundation/k12-architecture-kit/blob/904355737b09d59a594c9c9ed256726c48d553ec/unpackaged/config/trial/profiles/Admin.profile#L117

FYI @kdy618

Reply:
@MatthewBlanski , if I remove the %%%NAMESPACE%%%, I get an error when I create a new scratch org. Since it has the namespaced in front of it, it makes me think that it is calling/referring it to the src folder. Is that accurate?

Reply:
Namespace removed. Resolving.